modules/assim.batch/R/pda.load.data.R

```diff
@@ -81,6 +89,9 @@ load.pda.data <- function(settings, con) {
       
       inputs[[i]]$obs <- AMFo
       inputs[[i]]$par <- c(AMF.params$intercept, AMF.params$slopeP, AMF.params$slopeN)
+    }else{
+      inputs[[i]]$obs <- inputs[[i]]$data[colnames(inputs[[i]]$data) %in% inputs[[i]]$variable.name]
+      inputs[[i]]$par <- sd(unlist(inputs[[i]]$obs), na.rm = TRUE) # testing
```

Comment:
@mdietze here I pass the sd in the data and pass it along for calculating the likelihood

1) should I leave it this way or update it as part of the MCMC?
2) should I include a multiplicative bias term in the likelihood (see pda.define.llik.fn :point_up: )
3) should I use another likelihood all together for soil respiration?

![modelanddata](https://cloud.githubusercontent.com/assets/13728388/19120435/ba2623ac-8af0-11e6-8953-2a60defbf2df.png)
![modelvsdata](https://cloud.githubusercontent.com/assets/13728388/19120444/bcb53068-8af0-11e6-9d9b-15c1eb8a8732.png)
![resid](https://cloud.githubusercontent.com/assets/13728388/19120447/bee3cd36-8af0-11e6-9177-4a10920fccc5.png)